### PR TITLE
Feature/add guest picker change functionality

### PR DESCRIPTION
**Pull request by lighthurst**
Hi @justuskw @Pisotski @briansai, whoever is avail, please review whenever. See Trello ticket for visuals. Thank you. Back in town tomorrow woot woot.
https://trello.com/c/Phv85XlT/19-implement-guest-picker-functionality-m

### CODE REVIEW

File: client/src/components/GuestPicker.jsx [hunk 1]
```diff
@@ -17,6 +25,40 @@ export default class GuestPicker extends Component {
...
+      labelNum -= 1;
+    }
+    if (label !== 'Infants' && (guests - this.state[otherLabel]) > 0) {
+      guests -= 1;
+    }
+    this.setState({ guests, [stateLabel]: labelNum });
+  }
   handleOutsideClick(e) {
     if (this.props.guestDropDownActive) {
       if (this.dropDownRef.current.contains(e.target)) {
```
Comment (justuskw) [inline client/src/components/GuestPicker.jsx:64]: I see you had to do a outside click to close out the dropdown like I had to do for my popup. Good work!

File: client/src/components/GuestPickerDropDown.jsx [hunk 2]
```diff
@@ -4,31 +4,39 @@ import styled from 'styled-components';
...
+            label={row[0]}
+            secondaryLabel={row[1]}
+            value={row[2]}
+            handleAddGuests={handleAddGuests}
+            handleMinusGuests={handleMinusGuests}
+            totalGuests={totalGuests}
+            maxGuests={maxGuests}
+            key={`${row[0]} row key`}
+          />))
+      }
```
Comment (justuskw) [inline client/src/components/GuestPickerDropDown.jsx:26]: Nice refactor.

**Review by justuskw — APPROVED** [review]
Looks good.

**Comment by lighthurst** [comment]
Thanks @justuskw for the quick turnaround and compliment!

**Pull request merged**